### PR TITLE
Add deprecate warning for buildNative task

### PR DESCRIPTION
**Pull request by glefloch**
@gastaldi, here is a pull request for the documentation. I was thinking of adding all available native properties. Something like https://quarkus.io/guides/building-native-image#configuration-reference

WDYT? Should I update the other pull request with something like that?

### CODE REVIEW

File: docs/src/main/asciidoc/gradle-tooling.adoc [hunk 1]
```diff
@@ -264,6 +264,13 @@ Make sure to have `GRAALVM_HOME` configured and pointing to GraalVM version {gra
 Create a native executable using: `./gradlew buildNative`.
 A native executable will be present in `build/`.
 
+[WARNING]
+====
+The `buildNative` task has been deprecated in favor of task +
```
Comment (gastaldi) [inline docs/src/main/asciidoc/gradle-tooling.adoc:269]:
```suggestion
The `buildNative` task has been deprecated in favor of
```

**Review by gastaldi — CHANGES_REQUESTED** [review]
I added a minor suggestion, see if that makes sense. 

You can add the following snippet to the documentation if you want all available native properties as a table: 

```asciidoc
include::{generated-dir}/config/quarkus-native-pkg-native-config.adoc[opts=optional]
```

**Comment by glefloch** [comment]
@gastaldi thanks for the review, it totally makes sense. I updated the doc. 
Thanks for the snippet, I will add it in the other pull request.

**Review by gastaldi — APPROVED** [review]
LGTM

**Review by geoand — APPROVED** [review]

**Comment by andreas-eberle** [comment]
What was the reasoning behind the deprecation of the buildNative task? If I have a gradle project with many microservices, I was able to use the buildNative task to configure common properties for all of them in the parent gradle file. Now I'll have to it for every module/every microservice.

**Pull request merged**